data_prep/mp_prep_other_inputs_one_off.py

```diff
@@ -169,63 +292,79 @@ def mp_prep_other_inputs(tile_id_list, process):
+        if cn.NO_UPLOAD == False:
+            for i in range(0, len(output_dir_list)):
+                uu.upload_final_set(output_dir_list[i], output_pattern_list[i])
 
-        # Creates primary forest tiles
-        source_raster = primary_vrt
-        out_pattern = 'primary_2001'
-        dt = 'Byte'
-        if cn.count == 96:
-            processes = 45  # 45 processors = 650 GB peak
-        else:
-            processes = int(cn.count/2)
-        uu.print_log("Creating primary forest tiles with {} processors...".format(processes))
-        pool = multiprocessing.Pool(processes)
-        pool.map(partial(uu.mp_warp_to_Hansen, source_raster=source_raster, out_pattern=out_pattern, dt=dt), tile_id_list)
-        pool.close()
-        pool.join()
+    ##################################################################################################
 
+    if process == 'us_forest' or process == 'all':
 
-        ### Creates a combined IFL/primary forest raster
-        # Uses very little memory since it's just file renaming
-        if cn.count == 96:
-            processes = 60  # 60 processors = 10 GB peak
-        else:
-            processes = int(cn.count/2)
-        uu.print_log("Assigning each tile to ifl2000 or primary forest with {} processors...".format(processes))
-        pool = multiprocessing.Pool(processes)
-        pool.map(prep_other_inputs.create_combined_ifl_primary, tile_id_list)
-        pool.close()
-        pool.join()
+        uu.print_log('Pre-processing US forest age, group, and region category tiles')
 
+        #List of output directories and output file name patterns
+        output_dir_list = [
+                           cn.FIA_forest_group_processed_dir,
+                           cn.age_cat_natrl_forest_US_dir,
+                           cn.FIA_regions_processed_dir,
+        ]
+
+        output_pattern_list = [
+                               cn.pattern_FIA_forest_group_processed,
+                               cn.pattern_age_cat_natrl_forest_US,
+                               cn.pattern_FIA_regions_processed,
+        ]
+
+        # If the model run isn't the standard one, the output directory and file names are changed
+        if sensit_type != 'std':
+            uu.print_log('Changing output directory and file name pattern based on sensitivity analysis')
+            output_dir_list = uu.alter_dirs(sensit_type, output_dir_list)
+            output_pattern_list = uu.alter_patterns(sensit_type, output_pattern_list)
+
+        # A date can optionally be provided by the full model script or a run of this script.
+        # This replaces the date in constants_and_names.
+        # Only done if output upload is enabled.
+        if cn.RUN_DATE is not None and cn.NO_UPLOAD is not None:
+            output_dir_list = uu.replace_output_dir_date(output_dir_list, cn.RUN_DATE)
+
+        # Files to process: climate zone, IDN/MYS plantations before 2000, tree cover loss drivers, combine IFL and primary forest
```

Comment:
Update this list of files to process.

Reply:
Changes made in the following commit: 064f1d8211bb345593bb2eca6e424b5b216a7475